Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py

```diff
@@ -0,0 +1,1516 @@
+                    + ' with the job_id: ' + str(job_id),
+                    outputs={
+                        'Malwarebytes.Scan(val.Job_ID == obj.Job_ID)': {
+                            'Machine_ID': ids,
+                            'Job_ID': job_id
+                        }
+                    }
+                )
+            else:
+                return_error(message='Desktop Isolation action Failed for the Endpoint: ' + str(hostname), error='')
+        else:
+            return_error(message="Endpoint is not found in the Malwarebytes Cloud", error='')
+    else:
+        return_error(message="Please Enter IP or Hostname", error='')
+
+
+# Function to execute Deisolate Command
+def deisolate_endpoint(account_id, client_id, auth_token, ip, hostname):
+    # This is the call made initiating scan_and_report action
+    if ip:
+        ids = get_machine_id_ip(account_id, client_id, auth_token, ip)
+        if ids:
+            job_id = post_deisolate(account_id, client_id, auth_token, ids)
+            if job_id:
+                return_outputs(
+                    'Deisolation action has been successfully started on the Endpoint: ' + str(ip)
+                    + ' with the job_id: ' + str(job_id),
+                    outputs={
+                        'Malwarebytes.Scan(val.Job_ID == obj.Job_ID)': {
+                            'Machine_ID': ids,
+                            'Job_ID': job_id
+                        }
+                    }
+                )
+            else:
+                return_error(message='Deisolation action Failed for the Endpoint: ' + str(ip), error='')
+        else:
+            return_error(message="Endpoint is not found in the Malwarebytes Cloud", error='')
+
+    elif hostname:
+        ids = get_machine_id_hostname(account_id, client_id, auth_token, hostname)
+        if ids:
+            job_id = post_deisolate(account_id, client_id, auth_token, ids)
+            if job_id:
+                return_outputs(
+                    'Deisolation action has been successfully started on the Endpoint: ' + str(hostname)
+                    + ' with the job_id: ' + str(job_id),
+                    outputs={
+                        'Malwarebytes.Scan(val.Job_ID == obj.Job_ID)': {
+                            'Machine_ID': ids,
+                            'Job_ID': job_id
+                        }
+                    }
+                )
+            else:
+                return_error(message='Deisolation action Failed for the Endpoint: ' + str(hostname), error='')
+        else:
+            return_error(message="Endpoint is not found in the Malwarebytes Cloud", error='')
+    else:
+        return_error(message="Please Enter IP or Hostname", error='')
+
+
+# Function to execute List all endpoints Command
+def list_all_endpoints(account_id, client_id, auth_token, endpoint):
+    if endpoint == 'all':
```

Comment:
lets not duplicate code here, and use the conditions just for building the list of endpoint need to be output